api/v1alpha2/awscluster_types.go

```diff
@@ -49,6 +49,10 @@ type AWSClusterSpec struct {
 
 // AWSLoadBalancerSpec defines the desired state of an AWS load balancer
 type AWSLoadBalancerSpec struct {
+	// Ingress rules associated with the load balancer's Security Group associated with the load balancer/Infra
```

Comment:
The comment here should probably also state that 0.0.0.0/0 will be used if unspecified.